doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1077,6 +1079,9 @@ DFU libraries
 
 * Added a new DFU SMP target for the image update to an external MCU by using the MCUmgr SMP Client.
```

Comment:
Is this referring to a specific DFU library, or just a common thing within the DFU libraries?

Reply:
I did not add `* Added a new DFU SMP target for the image update to an external MCU by using the MCUmgr SMP Client."` this part if you are referring to that? or are you referring to something else?

Reply:
I've added this part as suggested
```

* :ref:`subsys_pcd` library:

  * Added function :c:func:`pcd_network_core_app_version` for reading peripheral CPU application version number.
  
 ```
 
 Should it be moved back to the PCD subsection? You suggested to remove it in a previous comment so I removed it.

Reply:
Yep, that is what I mean. I need to find out who wrote it then.

Reply:
That would be @jheiskan81

Reply:
last edited 82c0e739c7

Reply:
No worries @sigvartmh, no need to move anything anywhere. We are good. Just need to squeeze Juha Heiskanen about the earlier entry under DFU libraries.